lib/gollum/views/helpers.rb

```diff
@@ -2,31 +2,63 @@
 
 module Precious
   module Views
+    module RouteHelpers
+      ROUTES = {
+        'gollum' => {
+          :last_commit_info => 'last-commit-info',
+          :latest_changes => 'latest_changes',
+          :upload_file => 'uploadFile',
+          :create => 'create',
+          :delete => 'delete',
+          :edit => 'edit',
+          :pages => 'pages',
+          :history => 'history',
+          :rename => 'rename',
+          :preview => 'preview',
+          :compare => 'compare',
+          :search => 'search'
+        }
+      }
+
+      def self.parse_routes(routes, prefix = '')
+        routes.each do |name, path|
+          if path.respond_to?(:keys) then
+            self.parse_routes(path, "#{prefix}/#{name}")
+          else
+            define_method :"#{name.to_s}_path" do
+              "#{base_url}/#{prefix}/#{path}".gsub(/\/{2,}/, '/')
+            end
+          end
+        end
+      end
+
+      def self.included(base)
+        self.parse_routes(ROUTES)
+      end
+    end
+
     module SprocketsHelpers
       def self.included(base)
 
-        def helper_proc_with_options(method)
-          Proc.new do |args|
+        def sprockets_stylesheet_tag
+          lambda do |args|
             args = args.split(' ')
-            if args.size > 1 then
-              options = args[1..-1].join(' ')
-              options = YAML.safe_load("---\n#{options}\n", [Symbol])
-            end
-            options = options.respond_to?(:to_h) ? options.to_h : {}
-            options = options.inject({}){|memo,(k,v)| memo[k.to_sym] = v; memo}
-            send(method, args[0], options)
-          end  
+            name = args[0]
+            options = {:media => :all}
```

Comment:
Okay, this answers my earlier remark. 👍

Reply:
Yes, I didn't want to turn the stylesheet tag content into quasi-syntax like structures, and it turns out we only ever use `{:media => :all}`. If someone ever designs a mobile frontend we'll adapt. ;)